src/LoadStorage.js

```diff
@@ -0,0 +1,16 @@
+import { MyTasks } from './ClassCall.js';
+
+const myStorage = JSON.parse(localStorage.getItem('myArray'));
+export default () => {
+  if (myStorage !== null) {
+    document.addEventListener('DOMContentLoaded', () => {
+      myStorage.forEach((task) => {
+        MyTasks.retriveStorage(task.task, task.done, task.index);
```

Comment:
The function retrieveStorage() in ClassMaster.js only takes two parameters as it was written. Kindly make sure that your function calls are congruent with your function declartations